Adyen/Core/Core Protocols/Details.swift

```diff
@@ -7,14 +7,7 @@
 import Foundation
 
 /// Contains details supplied by a component. These details are used to initiate or complete a payment.
-public protocol Details: Encodable {}
-
-public extension Details {
-    
-    /// Provides a concrete encodable object to easily encode any `Details` conforming object.
-    var encodable: AnyEncodable { AnyEncodable(value: self) }
-    
-}
+public protocol Details: OpaqueEncodable {}
```

Comment:
I don't get what is the difference ? 🤔

Reply:
its just convenience, because `OpaqueEncodable ` has default implementation of the `var encodable`, so that types can adopt the protocol without needing to implement the property

Reply:
But `Details` also have default implementation for `var encodable: AnyEncodable { get }`?
And `Details` is the only protocol that implements it..